web/html/src/manager/maintenance/list/maintenance-windows-list.tsx

```diff
@@ -38,6 +38,11 @@ const MaintenanceWindowsList = (props: MaintenanceListProps) => {
         title={t("Maintenance") + " " + (type === "schedule" ? t("Schedules") : t("Calendars"))}
         icon="spacewalk-icon-schedule"
         buttons={buttons}
+        helpUrl="reference/schedule/maintenance-windows.html"
+        summary={t(
+          "Below is a list of Maintenance {0} available to the current user.",
+          type === "schedule" ? t("Schedules") : t("Calendars")
```

Comment:
Instead of this, I would recommend splitting this into two long translation strings. Otherwise the translation will be tricky to do in languages where the sentence order or logic is different.

Reply:
Yeah makes sense :+1: